packages/cli/CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Change Log
 All notable changes to the Zowe CLI package will be documented in this file.
 
-
 ## Recent Changes
 
+- BugFix: When using the `copy` command, when a target partitioned data set has a smaller record length than a source partitioned data set, it no longer stops the operation for subsequent members. The user can now view the affected members in a local file. [#2349] (https://github.com/zowe/zowe-cli/issues/2349)
```

Comment:
- For better readability, change the second "when" to "if": "When using the `copy` command, **if** a target partitioned data set ... ."
- Also, rephrase the sentence a bit: " ... a source partitioned data set, the operation for subsequent members **no longer stops**." OR: " ... a source partitioned data set, the operation for subsequent members **is no longer interrupted**."